services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/utils.py

```diff
@@ -35,39 +35,40 @@ class _RegistryNotReachableException(Exception):
     pass
 
 
+@retry(
+    wait=wait_fixed(1),
+    stop=stop_after_attempt(1),
```

Comment:
not sure I understand the point of using tenacity if retrial is not used..

Reply:
I agree but here i still did not change the design. 
I just moved it outside as a decorator. 
Next PRs will start doing changes in some of the design decisions.